src/kademlia/routing_table.cpp

```diff
@@ -57,15 +57,6 @@ POSSIBILITY OF SUCH DAMAGE.
 
 using boost::uint8_t;
 
-#if BOOST_VERSION <= 104700
-namespace boost {
-size_t hash_value(libtorrent::address_v4::bytes_type ip)
```

Comment:
I imagine this was used for the unordered set of IPs. does ip_set use an unordered_set keyed on union_address now? (I seem to recall you had two sets, which seems right, but isn't this still necessary then?)